src/Utils/obiDialogParser.ts

```diff
@@ -22,7 +22,7 @@ enum OBIRuleType {
 }
 
 export interface ObiDialogParserResult {
-    luMap: Map<string, string[]>
+    luMap: { [key: string]: string[] }
```

Comment:
Just curious, what's the benefit of changing from a `Map` to a dict here ?

Reply:
Matt's suggestion.

Reply:
Ah, from the previous review.  Got it.

@mattmazzola - is this a general best practice in JS/TS when the key is a string ?  Any details on why ?

Reply:
Well I would consider that a lazy answer. Glad that you changed because the suggestion came from me but in future I would hope you understand the reasoning of the suggestion and why  instead of just blindly changing code without understanding. That likely leads to more bugs.
It also doesn't answer the question "what's the benefit". 

Benefits are cleaner/simpler API and the follows expectation of reader.  You use `Maps` when they are required.  Such as when keys are not strings, or need to take advantage of some optimizations that data structure provides like the `size` property instead of `Object.keys()`.
If you use a `Map` reader thinks it was needed and then searching for case where it was required when there is no case. There is `Record<K,V>` type for similar syntax of typing and object that way also.